test/test_device_radix_sort.cu

```diff
@@ -904,9 +904,15 @@ void InitializeSolution(
             // Mask off unwanted portions
             if (num_bits < static_cast<int>(sizeof(KeyT) * 8))
             {
+#if CUB_IS_INT128_ENABLED
+                __uint128_t base = 0;
```

Comment:
Can `base` be of type `UnsignedBits` corresponding to `KeyT`?

Reply:
That's much better, thanks!